src/views/Start/Start.js

```diff
@@ -44,12 +49,13 @@ function Start() {
 
   const [sel_lang_text, set_sel_lang_text] = useState(
     localStorage.getItem('apphomelang')
-      ? localStorage.getItem('apphomelang') === 'ta'
+      ? localStorage.getItem('apphomelang') === 'hi'
```

Comment:
Is there any impact on a code for this line if we are storing this localstorage

Reply:
![Screenshot from 2023-07-15 00-02-06](https://github.com/Sunbird-ALL/all-ll-app/assets/137033511/9c6eeb6d-99f6-4d2f-a3db-77bf41715d29)


no we are not using this state anywhere